test/integration_test/specs/kubevirt-fedora-wait-first-consumer/fedora-vm-wait-first-consumer.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  labels:
+    app: fedora-vm
+    vm.kubevirt.io/template: fedora-server-small
+    vm.kubevirt.io/template.namespace: openshift
+  name: fedora-test-vm-wait-first-consumer
+  annotations:
+    cdi.kubevirt.io/storage.bind.immediate.requested: "true"
```

Comment:
We should not need this annotation. Does it not work without this?

Reply:
Removed this annotation and it seems to work without it as well.